web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -72,6 +73,19 @@
 	// Progress should be reset when the current memory or asset changes.
 	$: memoryIndex, assetIndex, reset();
 
+	const handleKeyDown = (e: KeyboardEvent) => {
+		if (e.key === 'ArrowRight' && canGoForward) {
+			e.preventDefault();
+			toNext();
+		} else if (e.key === 'ArrowLeft' && canGoBack) {
+			e.preventDefault();
+			toPrevious();
+		} else if (e.key === 'Escape') {
+			e.preventDefault();
+			goto(AppRoute.PHOTOS);
+		}
+	};
+
```

Comment:
So, it would be nice to use a switch but the code is way longer :/

```suggestion
	const handleKeyDown = (e: KeyboardEvent) => {
	  switch (e.key) {
	  case 'ArrowRight':
	    e.preventDefault();
	    if (canGoForward) {
	      toNext();
	    }
	    break;
	  case 'ArrowLeft':
	    e.preventDefault();
	    if (canGoBack) {
	      toPrevious();
	    }
	    break;
	  case 'Escape':
	    e.preventDefault();
	    goto(AppRoute.PHOTOS);
	    break;
	  }
	};
```

Reply:
Yeah, I thought about it, but as we have only 3 keys, I was not sure.
Anyway, I don't have preferences.